### PR TITLE
clean up function syntax

### PR DESCRIPTION
**Pull request by hermannm**
### Changes

- make syntax for `getLayout` more consistent
- remove unused props from about page `Template` component

### CODE REVIEW

File: frontend/src/lib/apolloClient.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export type PageProps = {
 
 let apolloClient: ApolloClient<NormalizedCacheObject> | undefined;
 
-function createApolloClient(ctx?: GetServerSidePropsContext): ApolloClient<NormalizedCacheObject> {
+const createApolloClient = (ctx?: GetServerSidePropsContext): ApolloClient<NormalizedCacheObject> => {
```
Comment (larwaa) [inline frontend/src/lib/apolloClient.ts:22]: These have been deliberately declared using `function` as I believe it is clearer to have a distinction between regular variables and functions. Arrow functions are nice, they make sense in combination with React.FC, but for regular old javascript code I think we should stick to functions. 

While there are more differences between them than the stylistic ones, none of them come into play here.

File: frontend/src/lib/emotion.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
-import createCache from "@emotion/cache";
+import createCache, { EmotionCache } from "@emotion/cache";
 
 const isBrowser = typeof document !== "undefined";
 
 // On the client side, Create a meta tag at the top of the <head> and set it as insertionPoint.
 // This assures that MUI styles are loaded first.
 // It allows developers to easily override MUI styles with other styling solutions, like CSS modules.
-export function createEmotionCache() {
```
Comment (larwaa) [inline frontend/src/lib/emotion.ts:8]: Applies here as well

File: frontend/src/lib/apolloClient.ts (hunk 1 above)
Comment (hermannm) [reply frontend/src/lib/apolloClient.ts]: Fair enough, changed back in 9f2d160.

**Review by simensandhaug — APPROVED** [review]
lgtm

**Pull request merged**